docker-compose.prod.yaml

```diff
@@ -0,0 +1,9 @@
+version: "3.8"
+services:
+  web:
+    build: .
+    image: ghcr.io/cfc-servers/cfc_heartbeat_server:latest.
+    ports:
+      - "8123:8123"
+    env_file:
+      - .env
```

Comment:
```suggestion
```
I'l all for being explicit, but `.env` is the default docker compose env file so we can omit this